website/src/content/concepts/oauth-scopes-and-user-permissions.mdx

```diff
@@ -30,6 +30,37 @@ Notice here how the OAuth Scopes are grouped by the two fields `view` and `manag
 
 This grouping determines the **mapping and relation between OAuth Scopes and user permissions**.
 
+# Additional OAuth Scopes
+
+Defining `oAuthScopes` in the Custom Application config allows using permissions limited to 1 unique pair (view/manage) specific to the Custom Application.
+
+For more granular permissions, for example, to allow the team access to only certain parts or functionality of the Custom Application, [additional OAuth Scopes](https://docs.commercetools.com/api/scopes) can be requested as part of various permission groups. 
```

Comment:
I think it would help if we can maybe provide a more "real-world" concrete example and how it can be solved by using multiple permission groups.